analyzer/src/analysis/value/interprocedural/sequential/function_fixpoint.cpp

```diff
@@ -76,7 +77,17 @@ FunctionFixpoint::FunctionFixpoint(
       _checkers(checkers),
       _exit_invariant(make_bottom_abstract_value(ctx)),
       _return_stmt(nullptr),
-      _logger(logger) {}
+      _logger(logger)
+{
+  if (_ctx.opts.trace_ar_statements) {
+    ar::TextFormatter formatter{};
+    auto& stream = analyzer::log::msg().stream();
+    stream << "\n>>>>>>>>>>>>>>\nEntering Interprocedural Sequential FunctionFixpoint for ";
```

Comment:
Arguably the API of our logger is not great, but anyway: you are not supposed to access the stream directly, except when there no other way (for instance, passing it to `formatter.format_header`). The reason is that the `LogMessage` does thing in its destructor.

```suggestion
    auto msg = analyzer::log::msg();
    msg << "\n>>>>>>>>>>>>>>\nEntering Interprocedural Sequential FunctionFixpoint for ";
```